session_2/student_registry/db/migrate/20210113102650_create_students.rb

```diff
@@ -0,0 +1,13 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :roll_number
+      t.string :branch
+      t.decimal :cgpa, :precision => 2, :scale => 2
```

Comment:
`precision: 2` is incorrect.

Precision defines the number of digits that can be stored and scale defines the number of digits after the decimal point.

For example, to store `9.45` you are storing 3 digits in total and 2 digits after the decimal point - so precision of 3 and scale of 2

Reply:
ok got it thanks